.github/workflows/conformance.yml

```diff
@@ -36,6 +36,6 @@ jobs:
       - name: Unpack sigstore-java distribution
         run: tar -xvf ${{ github.workspace }}/sigstore-conformance/build/distributions/sigstore-conformance-*.tar --strip-components 1
 
-      - uses: trailofbits/sigstore-conformance@0748d63c53810e36cc3f4bbe4114301080f0d844
+      - uses: trailofbits/sigstore-conformance@42e429d56d8d6afc71358e23e82ce6bdb7a0726d
```

Comment:
I'm a little confused by this update. I think we needed to pin to tags only or something. renovate might be a little aggressive on the updates.